### ISSUE TITLE
[Help] disable anchor animation on some element

### ISSUE THREAD

**Issue by azzamsa** [Pelican-Elegant/elegant#545]
## Mandatory Step

- [X] I am using latest production release of Elegant

## Recommended Step

- [X] I have attached link to my blog source code repository
- [X] I have reproduced my pelican configuration files

## I need help with

the new animation for anchor, break my custom css. The [docs](https://elegant.oncrashreboot.com/customize-style) regarding 'how to disable this behavior' is still not updated.

![image](https://user-images.githubusercontent.com/17734314/70372928-f004a580-1917-11ea-8f3b-f73fcbb8a543.png)

This is my approach, but no luck.

modified   static/css/links.css
``` diff
-article p:not(#list-of-translations):not(#post-share-links) a:not(.ampl),
-article ol a:not(.ampl),
-blockquote a:not(.ampl),
+article p:not(#list-of-translations):not(#post-share-links) a:not(.ampl):not(.badges),
+article ol a:not(.ampl):not(.badges),
+blockquote a:not(.ampl):not(.badges),
```

Thank you.

**Comment by talha131** [comment]
If I understand you correctly, you want to disable animation for social icons in the sidebar.

You need to override these

https://github.com/Pelican-Elegant/elegant/blob/master/static/css/sidebar-social.css

**Comment by azzamsa** [comment]
No, I make my own social bar.

![image](https://user-images.githubusercontent.com/17734314/70381952-12cda300-1986-11ea-91b0-62792fca9f66.png)


source: https://raw.githubusercontent.com/azzamsa/blog/source/content/pages/about.markdown
custom css: https://github.com/azzamsa/elegant/blob/myhack/static/css/custom.css
live: https://azzamsa.com/blog/about/

I've tried this too, but didn't work:

```diff
modified   static/css/links.css
@@ -1,6 +1,6 @@
-article p:not(#list-of-translations):not(#post-share-links) a:not(.ampl),
-article ol a:not(.ampl),
-blockquote a:not(.ampl),
+article p:not(#list-of-translations):not(#post-share-links) a:not(.ampl):not(.sicon),
+article ol a:not(.ampl):not(.sicon),
+blockquote a:not(.ampl):not(.sicon),
 article
   div.article-content
   ul:not(.articles-timeline):not(.related-posts-list)

```

**Comment by talha131** [comment]
I assume you have an issue with this dotted line

![](https://i.imgur.com/I0ztIVM.png)

Secondly, I notice that you have raw HTML in your markdown. Elegant relies on Pelican to compile and generate HTML. If a user has added raw HTML snippets in his markdown, then, of course, Elegant may fail.

This is the reason, I have been unable to reproduce your issue when I was fixing https://github.com/Pelican-Elegant/elegant/issues/533#issuecomment-559935701.

You need to add `border-bottom: none;`. Cause you have raw HTML, you can add it to the anchor element style or add it in custom.css.

**Comment by azzamsa** [comment]
> You need to add `border-bottom: none;`. Cause you have raw HTML, you can add it to the anchor element style or add it in custom.css.

I've tried to add `border-bottom: none;` to the raw html directly and to custom css, the link border didn't disappear. but at least it didn't cause blur as before (blur at the bottom of the circle).

![image](https://user-images.githubusercontent.com/17734314/70384268-00b42a80-19af-11ea-81da-cee13ddd22f6.png)

I also tried to add many `not` in links.css in order to ignore my element but that didn't work.

For now. I'm okay with this.
